tests/test_dsp_deconvolution.py

```diff
@@ -75,13 +75,28 @@ def test_output_length():
     assert res.n_samples == 7
 
 
-def test_output_sweep():
+@pytest.mark.parametrize("is_complex_sig1", [True, False])
+@pytest.mark.parametrize("is_complex_sig2", [True, False])
+def test_output_sweep(is_complex_sig1, is_complex_sig2):
     """test of flat output frequency response resulting from deconvolving
-    a sweep with the same sweep"""
-    sweep = pfs.exponential_sweep_time(44100, (100, 10000))
-    res = pf.dsp.dsp.deconvolve(sweep,
-                                sweep,
-                                freq_range=(1, 44100)).freq[0, 1000:-12500]
+       a sweep with the same sweep"""
+    fs = 44100
+    sweep_1 = pfs.exponential_sweep_time(fs, (100, 10000))
+    sweep_2 = pfs.exponential_sweep_time(fs, (100, 10000))
+
+    eval_range_start = 0
+    if is_complex_sig1:
+        sweep_1.fft_norm = 'none'
+        sweep_1.complex = is_complex_sig1
+        eval_range_start = int(fs/2)
+    if is_complex_sig2:
+        sweep_2.fft_norm = 'none'
+        sweep_2.complex = is_complex_sig2
+        eval_range_start = int(fs/2)
```

Comment:
This might be a bit more obvious:

```suggestion
    eval_range_start = sweep_1.find_nearest_frequency(0)
    if is_complex_sig1:
        sweep_1.fft_norm = 'none'
        sweep_1.complex = is_complex_sig1
        eval_range_start = sweep_1.find_nearest_frequency(0)
    if is_complex_sig2:
        sweep_2.fft_norm = 'none'
        sweep_2.complex = is_complex_sig2
        eval_range_start = sweep_2.find_nearest_frequency(0)
```